content/influxdb/v1.8/tools/api.md

```diff
@@ -146,6 +146,12 @@ Use this endpoint to check the health of your InfluxDB instance.
 curl -XGET "localhost:8086/health"
 ```
 
+##### /health endpoint responses
+| Response code | Health    | Message                        | Status |
+|:------------- |:------    |:-------                        | ------:|
+| 200           | Healthy   | `ready for queries and writes` | `pass` |
+| 503           | Unhealthy |                                | `fail` |
+
```

Comment:
As a practice, should we mention the "default" unexpected errors?

Reply:
I would say no, simply because there are a variety of errors you could/would get back with the default response.